hit-test.html

```diff
@@ -97,18 +99,19 @@
         document.querySelector('header').appendChild(xrButton.domElement);
 
         if (navigator.xr) {
-          navigator.xr.isSessionSupported('immersive-ar').then((supported) => {
+          navigator.xr.isSessionSupported('immersive-ar', sessionInit)
```

Comment:
`isSessionSupported()` shouldn't take any options other than the session mode.

Reply:
D'oh! Not sure why I thought it does...